libmproxy/contentviews.py

```diff
@@ -515,6 +515,24 @@ def get_by_shortcut(c):
             return i
 
 
+def add(obj):
+    for i in views:
+        if i.name == obj.name:
+            raise ContentViewException("Duplicate view: " + obj.name)
+
+    for prompt in view_prompts:
```

Comment:
This problem sucks. :neutral_face: 
This should be good enough for now, but at some point we need a "automatically find different prompt key" thingie.
Can we add a TODO here please?